packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -77,11 +76,7 @@ type TDispatchProps = {
 const trackSearchOperationDebounced = _debounce(searchQuery => trackSearchOperation(searchQuery), 1000);
 
 const Search = Input.Search;
-
-const AdaptedVirtualSelect = reduxFormFieldAdapter({
-  AntInputComponent: VirtSelect,
-  onChangeAdapter: option => (option ? (option as any).value : null),
```

Comment:
why is this property no longer defined when you use `reduxFormFieldAdapter`?

Reply:
As per my understanding, that adapter was for components other than Ant, where `onChange` was handled differently 

But since we're using Ant's Select, I don't think we need to override the `onChange` callback with `onChangeAdapter` prop

This might be the reason `onChangeAdapter` was omitted in [SearchResults](https://github.com/jaegertracing/jaeger-ui/blob/main/packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.tsx#L76) too while using it with Ant's Select